### PR TITLE
MineClonia compatibility - mcl_trees colors

### LINKED ISSUE #337: MineClonia trees not shown on the map
I think this is because of a rename of the mcl_core mod

maybe @corarona can help with this

### PR DESCRIPTION
**Pull request by nonfreegithub**
mcl_trees colors for MineClonia compatibility

fixes [#337](https://github.com/minetest-mapserver/mapserver/issues/337)

### CODE REVIEW

File: colors/mc2.txt [hunk 1]
```diff
@@ -187,18 +187,28 @@ mcl_comparators:comparator_off_sub 113 101 97
 mcl_comparators:comparator_on_comp 115 101 97
 mcl_comparators:comparator_on_sub 116 101 97
 mcl_core:acacialeaves 141 136 76
+mcl_trees:acacialeaves 141 136 76
```
Comment (corarona) [inline colors/mc2.txt]: the leaves follow these scheme:

mcl_trees:leaves_acacia
mcl_trees:leaves_oak
mcl_trees:leaves_dark_oak
mcl_trees:leaves_cherry_blossom

File: colors/mc2.txt [hunk 2]
```diff
@@ -187,18 +187,28 @@ mcl_comparators:comparator_off_sub 113 101 97
...
 mcl_core:birchleaves 47 97 47
+mcl_trees:birchleaves 47 97 47
 mcl_core:birchsapling 78 110 74
+mcl_trees:birchsapling 78 110 74
 mcl_core:birchtree 155 130 112
+mcl_trees:birchtree 155 130 112
 mcl_core:birchtree_bark 194 184 174
+mcl_trees:birchtree_bark 194 184 174
 mcl_core:birchwood 145 113 89
+mcl_trees:birchwood 145 113 89
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:wood_birch

etc.

File: colors/mc2.txt [hunk 3]
```diff
@@ -277,6 +298,7 @@ mcl_core:sandstonecarved 201 154 121
 mcl_core:sandstonesmooth 201 154 121
 mcl_core:sandstonesmooth2 201 154 121
 mcl_core:sapling 62 90 45
+mcl_trees:sapling 62 90 45
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:sapling_oak

for the normal sapling

File: colors/mc2.txt [hunk 4]
```diff
@@ -289,9 +311,13 @@ mcl_core:snow_7 221 229 234
 mcl_core:snow_8 221 229 234
 mcl_core:spruceleaves 56 104 68
 mcl_core:sprucesapling 55 72 53
+mcl_trees:sprucesapling 55 72 53
 mcl_core:sprucetree 97 71 57
+mcl_trees:sprucetree 97 71 57
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:tree_spruce

etc.

File: colors/mc2.txt [hunk 5]
```diff
@@ -307,23 +333,32 @@ mcl_core:stone_with_lapis 111 113 125
 mcl_core:stone_with_redstone 128 91 88
 mcl_core:stone_with_redstone_lit 128 91 88
 mcl_core:stripped_acacia 157 98 62
+mcl_trees:stripped_acacia 157 98 62
 mcl_core:stripped_acacia_bark 151 90 57
+mcl_trees:stripped_acacia_bark 151 90 57
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:bark_stripped_acacia

etc.

File: colors/mc2.txt [hunk 6]
```diff
@@ -307,23 +333,32 @@ mcl_core:stone_with_lapis 111 113 125
...
 mcl_core:stripped_jungle 170 113 83
 mcl_core:stripped_jungle_bark 175 117 86
 mcl_core:stripped_oak 153 115 80
 mcl_core:stripped_oak_bark 162 122 84
 mcl_core:stripped_spruce 107 79 64
+mcl_trees:stripped_spruce 107 79 64
 mcl_core:stripped_spruce_bark 109 81 67
+mcl_trees:stripped_spruce_bark 109 81 67
 mcl_core:tree 135 104 74
+mcl_trees:tree 135 104 74
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:tree_oak

File: colors/mc2.txt [hunk 7]
```diff
@@ -307,23 +333,32 @@ mcl_core:stone_with_lapis 111 113 125
...
 mcl_core:tree_bark 95 75 57
+mcl_trees:tree_bark 95 75 57
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:bark_oak

File: colors/mc2.txt [hunk 8]
```diff
@@ -307,23 +333,32 @@ mcl_core:stone_with_lapis 111 113 125
...
 mcl_core:vine 21 61 21
 mcl_core:water_flowing 26 70 90
 mcl_core:water_source 38 98 129
 mcl_core:wood 113 87 67
+mcl_trees:wood 113 87 67
```
Comment (corarona) [inline colors/mc2.txt]: mcl_trees:wood_oak

**Comment by corarona** [comment]
Added some comments not for every single one but as you can see they all follow the same logic in mcla

This was essentially the point of the switch to mcl_trees, to make the wood nodes all follow the same scheme and not be all over the place as they were with some wood types before.

**Comment by corarona** [comment]
You can see an exhaustive list here:

https://codeberg.org/mineclonia/mineclonia/src/branch/main/mods/ITEMS/mcl_trees/aliases.lua

**Comment by nonfreegithub** [comment]
fixed, I think

**Pull request merged**